config/application.rb

```diff
@@ -12,7 +12,8 @@ module SocialNetwork
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 6.0
-
+    config.api_only = false
+    config.session_store = :memory_store
```

Comment:
Is it required? `session_store` is cookies by default. Should work for html formats.